bindings/nodejs/lib/bindings.ts

```diff
@@ -100,15 +100,6 @@ const callWalletMethodAsync = (
         });
     });
 
-// Allow bigint to be serialized as hex string.
-//
-// Note:
-// Serializing `bigint` to a different format, e.g. to decimal number string
-// must be done manually.
-(BigInt.prototype as any).toJSON = function () {
-    return bigIntToHex(this);
-};
-
```

Comment:
Is it breaking change to move it? Was it directly accessible through bindings.ts?

Reply:
No it's not breaking, it just adds the function `toJSON` to `BigInt`, no matter where it's called from.